tests/unittest/test_fs_crashstorage.py

```diff
@@ -83,7 +85,8 @@ def nix_tmpdir(fn):
             + b'"payload": "multipart", '
             + b'"payload_compressed": "0", '
             + b'"payload_size": 645, '
-            + b'"throttle_rule": "accept_everything"'
+            + b'"throttle_rule": "accept_everything", '
+            + b'"user_agent": "wow"'
```

Comment:
I couldn't figure out how to use `ANY` here from `unittest.mock`, since this is a binary string, so I hardcoded a known header value. Open to alternatives if there's a better way.